packages/ui-app/src/Input.tsx

```diff
@@ -2,20 +2,23 @@
 // This software may be modified and distributed under the terms
 // of the ISC license. See the LICENSE file for details.
 
-import { BareProps } from './types';
+import { BareProps, UnlockI18n } from './types';
```

Comment:
Here surely it cannot be `UnlockI18N`? That sounds specific to locking - rather is needs to ahve a proper name to indicate use.

Reply:
Additionally, why is the string not rendered on the caller (via t(...) or anything else) and this prop just purely a string or React.ReactNode?

Reply:
Thanks. I've created type called `InputErrorMessage` and a component called `InputError` that accepts that type as a prop